### PR TITLE
app-backup/kfoldersync: version bump

### PR DESCRIPTION
**Pull request by Vindex17**

### CODE REVIEW

File: app-backup/kfoldersync/kfoldersync-3.1.0.ebuild [hunk 1]
```diff
@@ -25,7 +25,10 @@ RDEPEND="
 	$(add_frameworks_dep kitemviews)
 	$(add_frameworks_dep knotifications)
 	$(add_frameworks_dep kwidgetsaddons)
+	$(add_frameworks_dep kwindowsystem)
 	$(add_frameworks_dep kxmlgui)
+	$(add_qt_dep qtconcurrent)
+	$(add_qt_dep qtcore)
```
Comment (a17r) [inline app-backup/kfoldersync/kfoldersync-3.1.0.ebuild]: qtcore is already being pulled in by kde5.eclass, none of the inheriting ebuilds needs to set it ;)

Comment (Vindex17) [inline app-backup/kfoldersync/kfoldersync-3.1.0.ebuild]: Fixed

**Comment by kensington** [comment]
Is knotifications still needed?

**Comment by Vindex17** [comment]
No, I forgot to remove it from cmake. The change will come with next release

**Comment by kensington** [comment]
Thanks, picked as 304f67686bbd5503c43b6e30e63137b707cf2b23.

**Pull request closed without merging**